internal/motor/motor.go

```diff
@@ -23,15 +23,27 @@ type MotorNode struct {
 	// Account *at.BaseAccount
 }
 
-func CreateAccount(password, dscPubKey, psKey []byte) (*MotorNode, error) {
+func CreateAccount(password, dscPubKey []byte) (*MotorNode, error) {
 	m, err := setupDefault()
 	if err != nil {
 		return nil, err
 	}
 
 	// create Vault shards to make sure this works before creating WhoIs
 	vc := vault.New()
-	dscShard, pskShard, recShard, unusedShards, err := m.Wallet.CreateInitialShards()
+	deviceShard, sharedShard, recShard, unusedShards, err := m.Wallet.CreateInitialShards()
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: encrypt dscShard with dsc (i.e. webauthn)
+	dscShard := string(deviceShard)
+
+	// TODO: ecnrypt pskShard with psk (must be generated)
+	pskShard := string(sharedShard)
+
+	// password protect the recovery shard
+	pwShard, err := crypto.EncryptWithPassword(password, []byte(recShard))
```

Comment:
is it possible to generate the pwShard in a background process? this is going to see performance hits on lesser hardware with the current runtime configuration.